docs/user/metal/install_ipi.md

```diff
@@ -197,7 +197,7 @@ platform:
           username: admin
           password: password
         bootMACAddress: 00:11:07:4e:f6:71
-        hardwareProfile: default
+        hardwareProfile: default      
```

Comment:
Some whitespace snuck in, can you remove it? Otherwise LGTM, thanks!

Reply:
Removed